static/js/overlays.js

```diff
@@ -42,6 +42,19 @@ exports.recent_topics_open = function () {
     return open_overlay_name === "recent_topics";
 };
 
+exports.is_flatpickr_open = function () {
+    return $(".custom_user_field_value").hasClass("active");
+};
+
+exports.hide_flatpickr = function () {
+    const element = $(".flatpickr-calendar");
+    if (element.hasClass("open")) {
+        element.removeClass("open");
+    }
```

Comment:
You can safely `removeClass` without the `hasClass` check.

Should this do `current_flatpickr_instance.close()` (in `popovers.js`) instead?

Reply:
<code>current_flatpickr_instance</code> in (<code>popovers.js</code>) always throws as <code>undefined</code> value though the <code>flatpickr</code> is open .